### ISSUE TITLE
A11y_.NETCore_RecipesAppMyRecipesNew_NewRecipe_ScreenReader: Voice over is not announcing "Recipe Name" field as required field and not marked up with any chevron/symbol.

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/maui#15627]
### Description

[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-nikpap%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847543876&sdata=6wiDneF0tmnVt4CroW43rkfVbB%2BL%2BAAdn0nD7xR5hQM%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver"**

### GitHub Tags:
#A11yMAS; #A11ySev2; #A11yTCS; #DesktopApp; #BM_.NETCore_iOS_May23; #.NET Core; #WCAG1.3.1; #iOS; #iPhone; #Voiceover;

### Environment Details:​
Application Name: Rachel's Recipe
Operating System: iOS (iPhone 6s)
Screen Reader: Voiceover

### Repro Steps:

1. Install and open ".NET MAUI" app.
2. Swipe till "My Recipes" and double tap.
3. Swipe till "New" button and double tap.
4. "New Recipe" Screen will open.
5. Start Voice over.
6. Swipe till "Recipe Name" edit field and observe the issue.

### Actual Results:
Voice over is not announcing "Recipe Name" field as required field and not marked up with any chevron/symbol.

### Expected Results:
Voice over should announce "Recipe Name" field as required field and it should be marked up with any chevron/symbol.

### User Impact:
Screen reader users will not get information about the required field and will not understand whether it is required field.

### Attachment:

[Recording-20230619_123538.webm](https://github.com/dotnet/maui/assets/90901519/1bfe05ad-264c-44cf-8008-4f8f6b7a9215)

### Link to public reproduction project repository

Test

### Version with bug

Unknown/Other

### Last version that worked well

Unknown/Other

### Affected platforms

iOS

### Affected platform versions

iOS version 15.7.6

**Comment by PoojaNamde** [comment]
#Regressed:08-17-23; #Closed;

Verified the bug in latest environment.
Bug is fixed and does not repro. Hence closing the bug.
Please find the closing attachment
